docs/js/redux-saga.md

```diff
@@ -29,43 +29,47 @@ some examples! (read [this comparison](https://stackoverflow.com/questions/34930
 ## Usage
 
 Sagas are associated with a container, just like actions, constants, selectors
-and reducers. If your container already has a `sagas.js` file, simply add your
-saga to that. If your container does not yet have a `sagas.js` file, add one with
+and reducers. If your container already has a `saga.js` file, simply add your
+saga to that. If your container does not yet have a `saga.js` file, add one with
 this boilerplate structure:
 
 ```JS
 import { take, call, put, select } from 'redux-saga/effects';
 
-// Your sagas for this container
-export default [
-  sagaName,
-];
-
-// Individual exports for testing
-export function* sagaName() {
-
+// Root saga
+export default function* sagaName() {
+  // if necessary, start multiple sagas at once with `all` 
+  yield all([
+    helloSaga(),
```

Comment:
Since we don't have the multiple-injection problem anymore, maybe we can make the standard pattern to just inline `takeLatest` here - it's always been a little annoying defining multiple action-watching sagas.